### DISCUSSION TITLE
Is there a way to check other plain text files other than change lang mode?

### DISCUSSION THREAD

**Discussion by imaginariateatre** [valentjn/vscode-ltex#259] [Q&A] [answered]
Only Markdown and Latex are spellchecked by default, but is there a workaround to also check other plain-text files?

I write with VS Code in [fountain syntax](https://fountain.io/syntax), which is plain text but formatted to read as a movie script. The file ends ".fountain" and its Language Mode is Fountain. I need that, so I can use BetterFountain, a really helpful extension.

When I want to check the file with ltex, I change the language mode to Markdown, review, and after, change the mode again to keep writing. But it would be great if I could spell check while writing, otherwise the combing over all the "hide false positives" gets really tedious.

Thanks!

**Comment by valentjn** [comment, marked as answer]
I guess you should be able to use the command [`LTeX: Check Current Document`](https://valentjn.github.io/vscode-ltex/docs/commands.html#ltex-check-current-document) to trigger a manual check of unsupported documents.

LT<sub>E</sub>X will soon have better support for documents of arbitrary file types (i.e., with automatic checking) as there's now also another issue (#277) regarding this.

**Reply by valentjn** [reply]
@imaginariateatre I noticed LT<sub>E</sub>X already supports automatic checking of arbitrary file types. You just have to add the language identifiers (in your case probably `"fountain"`) to [`ltex.enabled`](https://valentjn.github.io/vscode-ltex/docs/settings.html#ltexenabled). I changed the description of that setting a bit to better reflect that activation events are not influenced. So in 10.0.0, there's a new command [`LTeX: Activate Extension`](https://valentjn.github.io/vscode-ltex/docs/commands.html#ltex-activate-extension), which you can run to activate the extension in case you work with unsupported file types.
